pjlib/src/pjlib-test/ioq_udp.c

```diff
@@ -180,6 +182,19 @@ static int compliance_test(const pj_ioqueue_cfg *cfg)
 	status=-20; goto on_error;
     }
 
+    ioque_name = pj_str((char*)pj_ioqueue_name());
+    if (pj_strncmp(&ioque_name, pj_cstr(&ioqueue_type, "epoll"), 5) == 0 ||
+        pj_strncmp(&ioque_name, pj_cstr(&ioqueue_type, "kqueue"), 6) == 0 ||
+        pj_strncmp(&ioque_name, pj_cstr(&ioqueue_type, "iocp"), 4) == 0) {
+      if (pj_ioqueue_get_os_handle(ioque) == NULL) {
+        PJ_LOG(1,(
+          THIS_FILE,
+          "...pj_ioqueue_os_handle() unexpectedly returned NULL"
+        ));
+        status=-21; goto on_error;
```

Comment:
Wouldn't it be better to have a constant than a magic number?

Reply:
The unit tests use magic numbers for errors :)
As long as they are unique, so hopefully we can identify which test that fails, it should be fine.